packages/ag-grid-styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/fusion-react-ag-grid-styles",
-  "version": "0.4.7",
+  "version": "29.3.4-next",
```

Comment:
Why next?

Reply:
So package will match ag-grid version. next will be slashed on release.